Update Open @ RIT Page and FOSS hours times

Yiroh opened the pull request:
Fix for Issue #188

MoralCode commented:
This is a good start! I think it would ideally be nice to remove the entire page (i.e. the file you edited in this PR) and instead change the link in the navbar to go straight to the openr.it website (or whatever the canonical URL for them is)

Although i worry about losing the history of the text above what you edited (about OPEN@RIT being approved in 2020 by provost granberg)

Yiroh commented:
Added the history back, and added a temporary history text file while I gather the history. Also in the first commit, I change the README to actually have steps on how to contribute and a link to the website this repository is for.

MoralCode commented:
> Added the history back, and added a temporary history text file while I gather the history. Also in the first commit, I change the README to actually have steps on how to contribute and a link to the website this repository is for.

Usually it's better to separate changes like this that are unrelated into another PR/branch to be considered separately.





> Added the history back,

Sounds good! Did the runbook not have any useful info at all on it? it seems like thats the overarching infrastructure manual for FOSS@RIT stuff, may be useful to link to

Yiroh commented:
> Sounds good! Did the runbook not have any useful info at all on it? it seems like thats the overarching infrastructure manual for FOSS@RIT stuff, may be useful to link to

That is true, it would not hurt to link to it, though maybe have a disclaimer that information on that website may be outdated?

Yiroh commented:
> Usually it's better to separate changes like this that are unrelated into another PR/branch to be considered separately.

I honestly didn't know it was adding to this until I was already at commit 3... it was just doing it automatically... I'll make branches on my forked repository so it doesn't happen again.

MoralCode commented:
> > Sounds good! Did the runbook not have any useful info at all on it? it seems like thats the overarching infrastructure manual for FOSS@RIT stuff, may be useful to link to
> 
> That is true, it would not hurt to link to it, though maybe have a disclaimer that information on that website may be outdated?

Would the disclaimer then be liable to become outdated itself though (i.e. if someone modernizes the runbook?) How far is too much? Idk

Id just say "heres the runbook/instructions for how to run things in case its useful" or something like that

Yiroh commented:
> Id just say "heres the runbook/instructions for how to run things in case its useful" or something like that

would that be fine to add into this PR?
In addition to that, changing the meeting place, time, and day of the week? I know those are subject to change every semester...

MoralCode commented:
> would that be fine to add into this PR?

Sure!

> In addition to that, changing the meeting place, time, and day of the week? I know those are subject to change every semester...

Idk how pedantic to be with this repo tbh. Larger projects will probably want to separate that into another PR  to keep things separate/organized (heck, maybe it can be automated somehow), but yeah idk if I wanna make you go through all that if the change is just small/simple. That said it may be worth seeing if that small change leads to a larger change, like  if the date is posted in several places (maybe excluding old blog posts?) unifying the date into one place like i did with the IRC link recentlyish (#178) may be a good PR to make separately

Yiroh commented:
> That said it may be worth seeing if that small change leads to a larger change, like if the date is posted in several places (maybe excluding old blog posts?) unifying the date into one place like i did with the IRC link recentlyish (#178) may be a good PR to make separately

So in the config.yml? That is currently where I did the changes...
![image](https://github.com/FOSSRIT/fossrit.github.io/assets/69987911/9d190058-8692-4d41-b788-f9f5c025e0f3)

MoralCode commented:
oh perfect, i didnt realize it was already modular like that

Yiroh commented:
Alright, committed those changes to this PR. Done adding on to this PR unless something needs to be resolved.

MoralCode commented:
is history.txt just notes or is that file intended to be a publicly viewable webpage?

Yiroh commented:
just notes for now

MoralCode commented on README.md:
```diff
@@ -13,10 +13,20 @@ This is the website for the Free and Open Source Software initiative at the [RIT
+1. Fork this repository!
+2. Make your desired changes to either add additional features or repair existing ones.
+3. Commit your changes to your forked repository.
+4. Push to your forked repository's main branch.
```
This generally isnt the recommended strategy. I would either say nothing or encourage people to make feature branches

Yiroh replied:
I agree. Changes will be made to reflect that strategy.

MoralCode commented on history.txt:
```diff
@@ -0,0 +1,40 @@
+
+
+Information Contribution Credit:
+axk4545
```
its not super clear which info was contributed but maybe this isnt as important

Yiroh replied:
It would be eventually difficult to credit everyone for their specific info when its the same info repeated in some cases? SJ, Michael Nolan, and Chris Baker will probably send me one big package without discerning who contributed what info to it...

MoralCode replied:
yeah, at that point it may be too much to keep track of who contributed what, especially for a somewhat factual listing of what happened when. also this isnt the focus of this PR so its probably fine

MoralCode commented on open-rit.md:
```diff
@@ -7,12 +7,9 @@ slug: open-rit
-Open@RIT is the new Open Programs Office for the University directed by [Professor Stephen Jacobs](https://www.rit.edu/directory/sxjics-stephen-jacobs).  It's 
-creation was approved by Provost Dr. Ellen Granberg at the end of June 2020 and placed under the Office of the Vice-President of Research, Dr. Ryne Raffaelle 
-at the beginning of August.  
+You can find the Open@RIT Website here: [Open@RIT](https://openr.it/). Open@RIT is the new Open Programs Office for the University directed by [Professor Stephen Jacobs](https://www.rit.edu/directory/sxjics-stephen-jacobs). It’s creation was approved by Provost Dr. Ellen Granberg at the end of June 2020 and placed under the Office of the Vice-President of Research, Dr. Ryne Raffaelle at the begin ...
```
i would make this start with "OPEN@RIT now has their own [website](https://openr.it)"

Yiroh replied:
Agreed, made the changes.

MoralCode commented on open-rit.md:
```diff
@@ -7,12 +7,9 @@ slug: open-rit
 
-This page on the FOSS student GitHub site has been created as a placeholder in the spirit of "release eary, release often."  An official website for Open@RIT has
-been requested and is in the production queue with the RIT web team.
+To learn more about what Open@RIT does, you can visit their summary page, [What We Do](https://openr.it/about/).
```
maybe the words "their summary page" should be linked instead?

Yiroh replied:
I agree, made these changes

Yiroh commented:
Made the changes you suggested, will make a feature branch for history from now on
